airflow/providers/common/sql/hooks/sql.py

```diff
@@ -348,7 +348,7 @@ def get_cursor(self):
         return self.get_conn().cursor()
 
     @staticmethod
-    def _generate_insert_sql(table, values, target_fields, replace, **kwargs):
+    def _generate_insert_sql(table, values, target_fields, replace, placeholder="%s", **kwargs):
```

Comment:
Perhaps this should be a class attribute instead? It’s hard to imagine a SQL backend would need to support multiple placeholder types, and we will probably need to do the same for e.g. `UPDATE` as well.

Reply:
Makes sense, and it would be the same behaviour as with other backend-specific parameters.